src/rpc/server_util.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_RPC_SERVER_UTIL_H
+#define BITCOIN_RPC_SERVER_UTIL_H
+
+#include <rpc/protocol.h>
+#include <rpc/request.h>
+
+class CBlockPolicyEstimator;
+class CConnman;
+class CTxMemPool;
+struct NodeContext;
+class PeerManager;
+
+/**
+ * Ensure* helpers
+ */
```

Comment:
Can remove this comment? The code is already self-documenting and one can see that all helpers in this section start with `Ensure`.